.github/workflows/next-build-image.yaml

```diff
@@ -27,6 +27,8 @@ on:
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
       - '[0-9]+.[0-9]+'
+    branches:
+      - 'release-**'
```

Comment:
Right now, the image from `main` is built daily (or on demand), not on every push; I guess because it takes almost an hour to build the images?
Would it make sense to do the same thing for the release branches?